src/schemas/HederaUtils.ts

```diff
@@ -249,3 +249,51 @@ export function lookupNFTTransfer(transaction: Transaction, tokenId: string): Nf
     }
     return result
 }
+
+export function decodeRedirectForTokenInput(inputArgs: string): ethers.Result {
+    const tokenAddress = `0x${inputArgs.slice(2, 42)}`
+    const encodedFunctionSelector = `0x${inputArgs.slice(42)}`
+    return new ethers.Result(tokenAddress, encodedFunctionSelector)
```

Comment:
`decodeRedirectForToken() `should check `inputArgs` validity and throw an exception if not.

Reply:
I believe inputArgs's validity has already been checked in the wrapper level (https://github.com/hashgraph/hedera-mirror-node-explorer/pull/993/files/1d89d9151b6f07eacd6b777048e5305aa48488cb#diff-968baa6ba16da48193fd76bbf3a04b15eaf0ed55fa5ba73be1173aea08da5c4cR308)